bananalyzer/__main__.py

```diff
@@ -206,12 +221,10 @@ def main() -> int:
 
     # Load the desired tests
     generator = TestGenerator()
-    tests = [
-        generator.generate_test(example, args.headless) for example in filtered_examples
-    ]
+    tests = [generator.generate_test(example) for example in filtered_examples]
 
     # Run the tests
-    return run_tests(tests, agent, args.pytest_args)
+    return run_tests(tests, agent, args.pytest_args, args.headless, args.turbo)
```

Comment:
should package new params in pytest_args

Reply:
these are not args that will be passed to pytest. dont think they should be combined imho

Reply:
oh true